gigasecond/gigasecond_test.spec.js

```diff
@@ -0,0 +1,23 @@
+import Gigasecond from './gigasecond';
+
+describe('Gigasecond', () => {
+
+  it('tells the gigasecond birth of a living person', () => {
+    let gs = new Gigasecond(new Date(1977, 5, 13));
+    let expectedDate = new Date(2009, 1, 19);
+    expect(gs.date()).toEqual(expectedDate);
+  });
+
+  xit('tells gigasecond birth of a future person', () => {
```

Comment:
It's not really important that it's prior or future. Also, we should use timestamps instead of dates, and have at least one test case that tests for a timestamp early in the day and one late in the day so we catch if people just calculate the number of days. We have one like this for ruby: https://github.com/exercism/xruby/blob/master/gigasecond/gigasecond_test.rb